chrome/script.browserify.js

```diff
@@ -86,15 +96,77 @@ function view() {
   ]);
 }
 
+function filterLogins(e) {
+  // remove executed search from input field
+  if (!fillOnSubmit && e.target.value.indexOf(domain) === 0) {
+    e.target.value = e.target.value.substr(domain.length);
```

Comment:
could you please explain to me the purpose of this piece?

It works weirdly...:
1. Open browserpass, it is in filter mode
2. Press Backspace to switch to search mode
3. Type "gith", press Enter
4. Press Backspace, everything is cleared (difficult to fix typo - not because of this code block, but still a valid issue)
5. Slowly type "gith" again, when you enter "h" of "gith" the field gets cleared again (because of this code block)

Reply:
It's a bug. Was half of a previous solution, which I have now removed.